### DISCUSSION TITLE
Reset filter not working when using verticals

### DISCUSSION THREAD

**Discussion by admsaxi** [microsoft-search/pnp-modern-search#3298] [Q&A]
I found an issue when using the pnp search which has multiple verticals, search results and search filters on one page. 

Let's say there are 2 verticals: documents and pages .
For each vertical we have a search results and search filter webpart. 

When I select a filter for a document, and then select the "pages vertical" the filter is still applied (but won't show any results because the applied filter won't have these filter values for a page). 
On the search results webpart, we enabled the "show selected filters" which also shows a button to reset the applied filters. 
Unfortunately, this button is not working when switching between verticals (the filter is not being resetted). 

Below I've added some screenshots:

**Step 1: apply a filter (content type = case):**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/148207966/d2f2ad85-731d-4aa5-8fe2-aed15a514e09)

**Step 2: switch to another vertical (Documents):**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/148207966/476e9b21-2acc-4e7b-9826-44c4ef5bdbc5)

**Step 3: try to remove/reset the applied filter (with console error from Edge):**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/148207966/e0f4a9ca-1602-4307-9d44-696be5c2256f)

Is this normal behaviour? 
Or is there a workaround to fix this? 
I'm using the latest release (4.9.3).

Many thanks!

**Comment by kasperbolarsen** [comment]
Thanks for reporting this one. It sounds like the same problem as this one, https://github.com/microsoft-search/pnp-modern-search/issues/3227
